### ISSUE TITLE
Support for latest docker-compose syntax

### ISSUE THREAD

**Issue by jongirard** [kontena/kontena#625]
I've been experimenting with Kontena, but ran into a block when I noticed that seemingly the 'latest' of the docker-compose.yml capability in the version 2 spec (https://blog.docker.com/2016/02/compose-1-6/) for defining services does not seem supported?

```
version: '2'

services:
  postgres:
    image: postgres:9.4.7

volumes:
```

Kontena init/deploy throws an undefined method `has_key?' for "2":String error.

**Comment by nevalla** [comment]
Unfortunately docker-compose.yml version 2 is not yet supported. We are pushing effort to support it on next releases. However at first phase volumes and networks might not be supported, but we are working hard to integrate them to Kontena as user-friendly way as possible.

**Comment by jakolehm** [comment]
Fixed in #696
